libpages/dns.go

```diff
@@ -63,14 +63,14 @@ const kbpRecordPrefix = "_keybase_pages."
 // _keybase_pages.meatball.gao.io   TXT "kbp=/keybase/public/songgao/meatball/"
 // _keybase_pages.song.gao.io       TXT "kbp=/keybase/private/songgao,kb_bot/blah"
 // _keybase_pages.blah.strib.io     TXT "kbp=/keybase/private/strib#kb_bot/blahblahb" "lah/blah/"
-// _keybase_pages.kbp.jzila.com     TXT "kbp=git-keybase://private/jzila,kb_bot/kbp.git"
+// _keybase_pages.kbp.jzila.com     TXT "kbp=git@keybase:private/jzila,kb_bot/kbp.git"
```

Comment:
I don't love this syntax; it doesn't match anything we're currently doing and it's not obvious.
Why isn't this `/keybase/private/jzila,kb_bot/.kbfs_autogit/public/jzila/kbp.git`?

Reply:
The checkout is now in `kb_bot` rather than `jzila,kb_bot`, but we can switch to the latter if desired I guess. One advantage for the `git@keybase` syntax though is it doesn't require user to remember the autogit path, and is shorter to type.

@strib any thoughts?

Reply:
Yeah I don't think we should expose the autogit path at this interface.  I suggested the `git@` syntax because it matches git remote syntax, and gives us the option in the future of adding other types of protocols.  I think we should hide any autogit implementation details away from the DNS TXT records.

Reply:
Then why is the `@keybase` necessary? Can it just be `kbp=git:private/jzila,kbp_bot/foo.git`?

Reply:
It's not necessary, it just matches well-known syntax like git's.  I dunno, it just seems like it plays more nicely -- maybe other providers would pop up that serve keybase pages and use the TXT field in the same way.

Reply:
I like `git@keybase:` slightly better over `git:` too, so will go with that for now. John let me know if you have a strong preference/reason for `git:`.

Reply:
Nope, I've been democratically overruled and defer to the will of The People.